plugins/scaffolder-backend/api-report.md

```diff
@@ -642,8 +648,12 @@ export interface RouterOptions {
   // (undocumented)
   logger: Logger;
   // (undocumented)
+  permissionApi?: PermissionEvaluator;
+  // (undocumented)
   reader: UrlReader;
   // (undocumented)
+  rules?: TemplatePermissionRuleInput[];
```

Comment:
I think that this is a bit generic for the `scaffolder-backend` `Router`. Let's go with `permissionRules` like the `CatalogBuilder`

https://github.com/backstage/backstage/blob/91cb7915994983167a5849bafe0badb9939ccf17/plugins/catalog-backend/src/service/CatalogBuilder.ts#L392-L399

Reply:
agree